### ISSUE TITLE
Triggered by overhead activity

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9932]
**Preferred term label:**
Triggered by overhead activity

**Synonyms**
Arms overhead

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Rotator cuff syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000683
- label: Triggered by overhead motions
- parents: [Triggered by (HP:0025204)](https://hpo.jax.org/app/browse/term/HP:0025204)
- pmid: PMID:23040553
- def: Applies to a sign or symptom that is induced by repeatedly performing overhead motions, such as painting a ceiling or working on objects located above the head, or some types of athletics.
- comment: This trigger can apply to rotator cuff syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9932
- synonyms: Triggered by overhead activity(NONE)
